en/docs/guides/login/saml-back-channel-logout.md

```diff
@@ -0,0 +1,26 @@
+# Configure SAML 2.0 Front Channel logout
+
+This page guides you through [configuring SAML back channel logout](TODO:insert-link-to-concept) for applications with WSO2 Identity Server. This will ensure that the user is logged out from all configured applications with a single logout request via one of the applications. 
+
+---
+
+This guide assumes you have your own applications. If you wish to try out this flow with sample applications, click the button below. 
+
+<a class="samplebtn_a" href="../../../quick-starts/saml-back-channel-logout" rel="nofollow noopener">Try it with the sample</a>
+
+---
+
+The instructions below guide you through configuring SAML 2.0 back channel Logout. 
+
+Register your applications with WSO2 Identity Server. Make sure you have a service provider configured in WSO2 Identity Server for every application that you are going to try this feature out on.  
+
+## Register your applications with WSO2 Identity Server
+
+TODO: dev-portal-fragment
+
+
+Once you have configured all your applications, access them in separate tabs in your browser. When you logout from one application, it should log you out from all the other configured applications. 
```

Comment:
in separate tabs --> on separate tabs

Reply:
logout --> log out 

same rule about verb vs noun as "login" and "login" apply here

Reply:
one of the application --> applications